WooCommerce/src/main/kotlin/com/woocommerce/android/ui/moremenu/MoreMenuFragment.kt

```diff
@@ -194,13 +180,32 @@ class MoreMenuFragment : TopLevelFragment() {
         )
     }
 
+    private fun openGoogleForWooWebview(url: String, canAutoLogin: Boolean) {
+        if (canAutoLogin) {
+            openInAuthBrowser(url)
+        } else {
+            openInExitAwareWebview(url)
+        }
+    }
+
+    private fun openInExitAwareWebview(url: String) {
+        findNavController().navigateSafely(
+            NavGraphMainDirections.actionGlobalExitAwareWebViewFragment(
+                urlToLoad = url,
+                urlsToTriggerExit = arrayOf(), // todo-11917: Replace with the right success URL
+                title = getString(R.string.more_menu_button_google),
+                urlComparisonMode = ExitAwareWebViewViewModel.UrlComparisonMode.PARTIAL
+            )
+        )
+    }
+
     private fun openInAuthBrowser(url: String) {
         findNavController().navigateSafely(
             NavGraphMainDirections.actionGlobalWPComWebViewFragment(
                 urlToLoad = url,
-                urlsToTriggerExit = arrayOf(),
+                urlsToTriggerExit = arrayOf(), // todo-11917: Replace with the right success URL
                 title = getString(R.string.more_menu_button_google),
-                clearCache = true
+                urlComparisonMode = ExitAwareWebViewViewModel.UrlComparisonMode.PARTIAL
```

Comment:
This should be `WPComWebViewViewModel.UrlComparisonMode.PARTIAL` instead? 
```suggestion
                urlComparisonMode = WPComWebViewViewModel.UrlComparisonMode.PARTIAL
```

Reply:
Good catch, thanks!